components/textfield/index.css

```diff
@@ -143,64 +148,106 @@ governing permissions and limitations under the License.
-    padding-inline-end: calc(var(--spectrum-textfield-padding-x-adjusted) + var(--spectrum-icon-alert-medium-width) + var(--spectrum-textfield-icon-margin-left));
+  .spectrum-Textfield.is-invalid & {
+    padding-inline-end: calc(var(--spectrum-textfield-padding-x) + var(--spectrum-icon-alert-medium-width) + var(--spectrum-textfield-icon-margin-left));
+  }
 
-    [dir="rtl"] & {
-      background-position: calc(var(--spectrum-textfield-padding-x-adjusted)) calc(calc(var(--spectrum-textfield-height) / 2) - calc(var(--spectrum-icon-alert-medium-height) / 2));
-    }
+  .spectrum-Textfield--multiline & {
+    block-size: var(--spectrum-textfield-multiline-height);
+    min-block-size: var(--spectrum-textfield-multiline-min-height);
+    padding: var(--spectrum-textfield-multiline-padding-top) var(--spectrum-textfield-multiline-padding-x) var(--spectrum-textfield-multiline-padding-bottom) calc(var(--spectrum-textfield-multiline-padding-x) - 1px);
+
+    /* Remove the default vertical scrollbar for textarea in IE. */
+    overflow: auto;
   }
 
-  &.is-valid {
-    background-size: var(--spectrum-icon-checkmark-medium-width) var(--spectrum-icon-checkmark-medium-width);
-    background-position: calc(100% - var(--spectrum-textfield-padding-x-adjusted)) calc(calc(var(--spectrum-textfield-height) / 2) - calc(var(--spectrum-icon-checkmark-medium-height) / 2));
-    padding-inline-end: calc(var(--spectrum-textfield-padding-x-adjusted) + var(--spectrum-icon-checkmark-medium-width) + var(--spectrum-textfield-icon-margin-left));
+  .spectrum-Textfield--quiet & {
+    border-radius: var(--spectrum-textfield-quiet-border-radius);
+    border-block-start-width: 0;
+    border-block-end-width: var(--spectrum-textfield-quiet-border-size);
+    border-inline-width: 0;
 
-    [dir="rtl"] & {
-      background-position: calc(var(--spectrum-textfield-padding-x-adjusted)) calc(calc(var(--spectrum-textfield-height) / 2) - calc(var(--spectrum-icon-checkmark-medium-height) / 2));
-    }
+    /* removes the side padding to align the text properly */
+    padding-inline-start: var(--spectrum-textfield-quiet-padding-x);
+    padding-inline-end: var(--spectrum-textfield-quiet-padding-x);
+
+    /* Treat all quiet inputs and textareas the same */
+    resize: none;
+    overflow-y: hidden;
   }
 }
 
-.spectrum-Textfield--multiline {
-  block-size: var(--spectrum-textarea-height);
-  min-block-size: var(--spectrum-textarea-min-height);
-  padding-block: var(--spectrum-textarea-padding-top-adjusted) var(--spectrum-textarea-padding-bottom-adjusted);
-  padding-inline: var(--spectrum-textarea-padding-x-adjusted) var(--spectrum-textarea-padding-x-adjusted);
+.spectrum-Textfield-validationIcon {
+  /* TODO: Confirm if this is ok for the validation icon sizing
+    Note that the sizes are a bit different when compared with old background icons(more noticable for checkmark)
+  */
```

Comment:
This Todo was carried over from the RSP code. @LFDanLu can you comment on it?

Reply:
Ah, I think this was a super old TODO when I first changed TextField and was testing against the wrong icons heh. Go ahead and remove the TODO

Reply:
Ok thanks!